examples/with-vite-react/pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '9.0'
+lockfileVersion: '6.0'
```

Comment:
Just flagging that this is the correct lockfile verison for the pnpm version. Previous one was incorrect